tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -321,6 +324,68 @@ func TestGlobalCheckAndReport(t *testing.T) {
 	}
 }
 
+func TestTcpMetrics(t *testing.T) {
+	if err := replaceRouteRule(tcpDbRule); err != nil {
+		t.Fatalf("Could not update reviews routing rule: %v", err)
+	}
+	defer func() {
+		if err := deleteRouteRule(tcpDbRule); err != nil {
+			t.Fatalf("Could not delete reviews routing rule: %v", err)
+		}
+	}()
+	allowRuleSync()
+
+	if err := visitProductPage(productPageTimeout, http.StatusOK); err != nil {
+		t.Fatalf("Test app setup failure: %v", err)
+	}
+	allowPrometheusSync()
+
+	glog.Info("Successfully sent request(s) to /productpage; checking metrics...")
+
+	promAPI, err := promAPI()
+	if err != nil {
+		fatalf(t, "Could not build prometheus API client: %v", err)
+	}
+	query := fmt.Sprintf("tcp_bytes_sent{destination_service=\"%s\"}", fqdn("mongodb"))
+	t.Logf("prometheus query: %s", query)
+	value, err := promAPI.Query(context.Background(), query, time.Now())
+	if err != nil {
+		fatalf(t, "Could not get metrics from prometheus: %v", err)
+	}
+	glog.Infof("promvalue := %s", value.String())
+
+	got, err := vectorValue(value, map[string]string{})
```

Comment:
shouldn't there be a subtraction of the previous value here too ?
(maybe that could be a function to delta the stats - or I missed a refactor and it already does that ?)